src/app/pokemon-list/pokemon-list.component.html

```diff
@@ -10,14 +10,17 @@
   ></mat-paginator>
 </div>
 
-@if (pokemons.isLoading()) {
+@if (allPokemons.isLoading()) {
 <p class="py-5 text-center text-muted fs-5" style="user-select: none">
   Loading Pokémon list...
 </p>
-} @else {
+} @else { @if (!pokemons().length) {
+<app-pokemon-error />
+}
```

Comment:
Having empty list is not an error. It can be an empty search. 

On top of it it's not handled here. The `@for` control flow has something called `@empty` to handle empty searches.